src/pages/account-menu/in-an-app-bar/index.tsx

```diff
@@ -0,0 +1,336 @@
+                                        classes={{ dot: classes.badge }}
+                                    >
+                                        {<Avatar alt="Chima Thabani" src={avatarImage} />}
+                                    </Badge>
+                                }
+                                menuGroups={menuGroupItems}
+                                menuTitle={avatarTitle}
+                                menuSubtitle={avatarSubtitile}
+                                MenuProps={{
+                                    anchorOrigin: {
+                                        vertical: 'bottom',
+                                        horizontal: 'right',
+                                    },
+                                    transformOrigin: {
+                                        vertical: 'top',
+                                        horizontal: 'right',
+                                    },
+                                    classes: { paper: classes.paper },
+                                }}
+                            />
+                        </Toolbar>
+                    </AppBar>
+                    {/* Text Menu Example */}
+                    <AppBar position="static" color="inherit" className={classes.appBar}>
+                        <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                            <IconButton color={'inherit'} edge={'start'}>
+                                <MenuIcon />
+                            </IconButton>
+                            <div className={classes.textContainer}>
+                                <Typography variant={'h6'} noWrap>
+                                    Text Menu
+                                </Typography>
+                                <Typography variant={'body1'} noWrap>
+                                    Calling Out the User Name
+                                </Typography>
+                            </div>
+                            <Spacer />
+                            <UserMenu
+                                onClick={toggleChip}
+                                avatar={
+                                    <Chip
+                                        variant="outlined"
+                                        icon={<AccountCircle style={{ color: colors.gray[500], marginLeft: '4px' }} />}
+                                        label="Admin"
+                                        rightIcon={
+                                            chipToggled ? (
+                                                <ExpandLessIcon fontSize={'small'} />
+                                            ) : (
+                                                <ExpandMoreOutlinedIcon fontSize={'small'} />
+                                            )
+                                        }
+                                        style={{ backgroundColor: chipToggled ? colors.white[500] : colors.white[50] }}
```

Comment:
This background color here overrides the hover styles that you would normally get on the chip. You might need to convert the inline style to a class so that you can add in the 'hover' styles.

Reply:
Do I need to create new prop `className` in Chip component to achieve this?

Reply:
If you define the Chip as a React.FC, it will automatically get the className prop:

```tsx
export const Chip: React.FC<ChipProps> = (props) => { ... }
```